redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -8,8 +8,8 @@ import scala.util.matching.Regex
 import zio.Chunk
 import zio.duration.Duration
 
-sealed trait Input[-A] {
-  private[redis] def encode(data: A): Chunk[RespValue.BulkString]
+trait Input[-A] {
```

Comment:
Please leave it sealed.

Reply:
Ok, add redisencoder, redisdecoder according to #214